Formula/trimage.rb

```diff
@@ -28,6 +28,8 @@ def install
   end
 
   test do
+    # Set QT_QPA_PLATFORM to minimal to avoid error "qt.qpa.xcb: could not connect to display"
+    ENV["QT_QPA_PLATFORM"] = "minimal" unless OS.mac?
```

Comment:
I think we can still run the full test on a local linux machine. Can you safeguard this with the HOMEBREW_GITHUB_ACTIONS env variable as this only applies to our CI?

Reply:
Yeah I think we can use the same conditions we use to skip these tests, namely `if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]`.